charters/IngestSupport/charter.md

```diff
@@ -0,0 +1,104 @@
+# Ingest Support
+
+## Description
+
+The Ingest Support team is the liaison between the Data Coordination Platform (DCP) and labs contributing data to the Human Cell Atlas (HCA). This team closely collaborates with each data contributor to:
+
+*	Working with data contributors who are generating new data types to develop a deep understanding of their data
+* Assisting data contributors to describe their experiments in a manner that supports finability, reusability and reproducability with in the DCP ecosystem
+* Helping data contributors to submit their data to the DCP, resolving any emerging issues during the process
+
+## Definitions
+
+*Data contributor* - any user who uploads data and metadata into the DCP
+
+*Data wrangler* - a member of the ingest support team who assists to data contributors wanting to upload data into the DCP
+
+*Submission* - the grouping of data and metadata that makes sense to a data contributor. For example, a data contributor might typically upload all the data at once for a publication they have in preparation. This grouping does not necessarily correspond to how data is stored or displayed in the DCP and how it should be displayed to a data consumer
```

Comment:
I would remove _typically_ as I don't think we want to claim to know how users likely will act. Better to just illustrate with an example of what someone _might_ do.

Reply:
rephrased